app/src/main/java/org/eyeseetea/malariacare/data/database/converts/OptionConverterFromDomainModelToDB.java

```diff
@@ -0,0 +1,21 @@
+package org.eyeseetea.malariacare.data.database.converts;
+
+
+import com.raizlabs.android.dbflow.annotation.NotNull;
+
+import org.eyeseetea.malariacare.data.database.model.OptionDB;
+import org.eyeseetea.malariacare.domain.boundary.converters.IConverter;
+import org.eyeseetea.malariacare.domain.entity.Option;
+
+public class OptionConverterFromDomainModelToDB implements IConverter<Option, OptionDB> {
+
+    @NotNull
+    @Override
+    public OptionDB convert(@NotNull Option domainModel) {
```

Comment:
Looks like you're following the visitor pattern here to make the conversion. If this is the case, we already have an standard naming convention for this that you can use. Check what's under sync -> exporter -> xxxVisitor. I think all that has to do with the IConverter could be adapted to the same naming convention

Reply:
:+1:

Reply:
@ifoche there also a similar name convention **xxxMaper** in the package *data.mappers* which one should I use?

Reply:
I've reviewed those mappers and I think your base implementation is closer to the visitors one, so I guess it would be much easier for everybody to understand it if you use the Visitor naming standard. However, maybe we can talk about which standard to use in our next meeting, so in the future, we're clear on which standard to follow

Reply:
:+1: